### ISSUE TITLE
Rent Fallhaven bed with not enough gold

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#287]
```
You should not be able to rent the room. It allows you to rent the room and you 
end up with a negative amount of gold.

More info: http://andorstrail.com/viewtopic.php?f=3&t=1236

Thanks to Grävling for finding the bug.

```

Original issue reported on code.google.com by `sdeva...@gmail.com` on 6 Dec 2011 at 2:14

**Comment by GoogleCodeExporter** [comment]
```
Thank you Grävling for finding it!
This should now be fixed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:47
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
